test/security/WSSecurityCert.js

```diff
@@ -54,6 +54,7 @@ describe('WSSecurityCert', function () {
     xml.should.containEql('<Signature xmlns="http://www.w3.org/2000/09/xmldsig#">');
     xml.should.containEql('<wsse:SecurityTokenReference');
     xml.should.containEql('<wsse:Reference URI="#' + instance.x509Id);
+    xml.should.containEql('<KeyInfo Id="' + instance.keyInfoId);
```

Comment:
I think test should check both. opening tag, like you already do, but closing tag `</KeyInfo>` as well.

Reply:
added starting and ending tag checks in the latest commit.